### PR TITLE
script.sh: `reinstall` / `uninstall`

### PR DESCRIPTION
**Pull request by commitay**
Refs: https://github.com/caskroom/homebrew-cask/pull/38547

`reinstall`: this allows us to test the install for Casks that are already installed in the Travis image.

`uninstall`: Travis doesn't seem to have any issues with `launchctl` or `quit`.

> https://travis-ci.org/caskroom/homebrew-cask/builds/281529752#L384

/cc @reitermarkus 

Edit: Changed this to run in a loop, easier to read (install/uninstall grouped by Cask) and avoids Cask [conflicts](https://travis-ci.org/caskroom/homebrew-cask/builds/281252332#L391).

### CODE REVIEW

File: ci/travis/script.sh [hunk 1]
```diff
@@ -22,7 +22,11 @@ elif [[ ${#modified_casks[@]} -gt 0 ]]; then
   run brew cask _audit_modified_casks "${TRAVIS_COMMIT_RANGE}"
   run brew cask style "${modified_casks[@]}"
   if [[ ${#modified_casks[@]} -le 10 ]]; then
-    run brew cask install "${modified_casks[@]}"
+  	for cask in "${modified_casks[@]}";
+    do
```
Comment (reitermarkus) [inline ci/travis/script.sh]: Move this to the same line as `for`.

File: ci/travis/script.sh [hunk 2]
```diff
@@ -22,7 +22,11 @@ elif [[ ${#modified_casks[@]} -gt 0 ]]; then
   run brew cask _audit_modified_casks "${TRAVIS_COMMIT_RANGE}"
   run brew cask style "${modified_casks[@]}"
   if [[ ${#modified_casks[@]} -le 10 ]]; then
-    run brew cask install "${modified_casks[@]}"
+  	for cask in "${modified_casks[@]}";
```
Comment (reitermarkus) [inline ci/travis/script.sh]: Indentation is off by one here.

File: ci/travis/script.sh [hunk 3]
```diff
@@ -22,7 +22,11 @@ elif [[ ${#modified_casks[@]} -gt 0 ]]; then
...
+    do 
+    run brew cask reinstall "${cask}";
+    run brew cask uninstall "${cask}";
```
Comment (reitermarkus) [inline ci/travis/script.sh]: Indent these one level more than `for`.

File: ci/travis/script.sh (hunk 2 above)
Comment (commitay) [reply ci/travis/script.sh]: I thought I had already fixed this, I've been editing a copy and pushing it to another repo. 😆

File: ci/travis/script.sh (hunk 3 above)
Comment (commitay) [reply ci/travis/script.sh]: Done.

File: ci/travis/script.sh [hunk 4]
```diff
@@ -22,7 +22,10 @@ elif [[ ${#modified_casks[@]} -gt 0 ]]; then
   run brew cask _audit_modified_casks "${TRAVIS_COMMIT_RANGE}"
   run brew cask style "${modified_casks[@]}"
   if [[ ${#modified_casks[@]} -le 10 ]]; then
-    run brew cask install "${modified_casks[@]}"
+    for cask in "${modified_casks[@]}"; do
+      run brew cask reinstall "${cask}";
```
Comment (reitermarkus) [inline ci/travis/script.sh]: The semi-colon here is also not needed. 😉

Comment (commitay) [reply ci/travis/script.sh]: Fixed.

**Review by reitermarkus — APPROVED** [review]

**Pull request merged**